lms/templates/bookmark_button.html

```diff
@@ -0,0 +1,11 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%page args="bookmark_id, is_bookmarked" />
+
+<div class="bookmark-button-wrapper">
+  <button class="btn bookmark-button ${"bookmarked" if is_bookmarked else "un-bookmarked"}"
+    aria-pressed="${"true" if is_bookmarked else "false"}"
+    data-bookmark-id="${bookmark_id}">
+     <span class="sr bookmark-sr">${_("Click to remove") if is_bookmarked else _("Click to add")}</span>&nbsp;
```

Comment:
@srpearce The text here is also for screen readers, this would be `Click to remove Bookmark` or `Click to add Bookmark` , The `Bookmark` text would be the text of actual button.